expirationd.lua

```diff
@@ -102,6 +102,11 @@ local function default_do_worker_iteration(task)
     local checked_tuples_count = 0
     local vinyl_checked_tuples_count = 0
     if task.atomic_iteration then
+        -- Before starting the transaction,
+        -- since the batch transaction can be long.
```

Comment:
Sorry, but I don't understand that "batch transaction" means.

Reply:
Can I replace it with transaction per batch?

Reply:
why not just "transaction"?

Reply:
Yeah, we can write just a transaction)

Reply:
Did this:
```diff
@@ -103,7 +103,7 @@ local function default_do_worker_iteration(task)
     local vinyl_checked_tuples_count = 0
     if task.atomic_iteration then
         -- Before starting the transaction,
-        -- since the batch transaction can be long.
+        -- since the transaction can be long.
         if task.worker_canceled then
             return true
         end
```